artifactory/commands/transferfiles/status.go

```diff
@@ -47,15 +47,9 @@ func ShowStatus() error {
 
 func addOverallStatus(stateManager *state.TransferStateManager, output *strings.Builder, runningTime string) {
 	windows := coreutils.IsWindows()
-	runningForTabs := 2
-	runningTimeString := "  " + runningTime
-	if windows {
-		runningForTabs = 3
-		runningTimeString = runningTime
-	}
```

Comment:
Just making sure you tested this on Windows after applying this change. If you did, all good.